packages/tasks_panel/src/commands/duplicate-task.ts

```diff
@@ -0,0 +1,39 @@
+import { ConfigurationTarget, Uri, window, workspace } from "vscode";
+import { ConfiguredTask } from "@sap_oss/task_contrib_types";
+import { getLogger } from "../logger/logger-wrapper";
+import { TaskTreeItem } from "../view/task-tree-item";
+import { messages } from "../i18n/messages";
+import { cleanTasks } from "../utils/ws-folder";
+import { cloneDeep, filter, find, map } from "lodash";
+
+export async function duplicateTask(treeItem: TaskTreeItem): Promise<void> {
+  try {
+    if (treeItem.command?.arguments === undefined) {
+      return;
+    }
+    const task = treeItem.command.arguments[0];
```

Comment:
is it possible that arguments would be empty array?

Reply:
not, it might exist. else I prefer the exception thrown and displayed